build.rs

```diff
@@ -206,14 +206,17 @@ fn static_lib_url() -> String {
     env::var("RUSTY_V8_MIRROR").unwrap_or_else(|_| default_base.into());
   let version = env::var("CARGO_PKG_VERSION").unwrap();
   let target = env::var("TARGET").unwrap();
+
+  // Note: we always use the release build on windows.
   if cfg!(target_os = "windows") {
-    // Note: we always use the release build on windows.
-    format!("{}/v{}/rusty_v8_release_{}.lib", base, version, target)
-  } else {
-    let profile = env::var("PROFILE").unwrap();
-    assert!(profile == "release" || profile == "debug");
-    format!("{}/v{}/librusty_v8_{}_{}.a", base, version, profile, target)
+    return format!("{}/v{}/rusty_v8_release_{}.lib", base, version, target);
   }
+  // Use v8 in release mode unless $V8_FORCE_DEBUG=true
+  let profile = match env::var("V8_FORCE_DEBUG").unwrap_or_default() == "true" {
```

Comment:
I know there are some who disagree but I'm of the opinion that binary env vars like this only need to be checked if they are set. That is we don't handle the case `V8_FORCE_DEBUG=0`, because no one would do that.. but we do easily handle `V8_FORCE_DEBUG=1` (common) as well as `V8_FORCE_DEBUG=true` (less common)
```rust
let profile = if env::var("V8_FORCE_DEBUG").is_some() { "debug" } else { "release "};
```

Reply:
I'm one of those people that think that "" and "0" and "false" should turn the flag off. 

But given that rusty_v8 is a Rust project, we should follow rust/cargo conventions.

Reply:
This is handled, it now acts like a "standard" env boolean and matches `true|1|yes` as truthy, everything else is false